docs/content/installation/csi_ug_requirements.md

```diff
@@ -27,43 +27,69 @@ For IBM Cloud® Satellite users, see [cloud.ibm.com/docs/satellite](https://clou
 
 -   Be sure that multipathing is installed and running.
 
-Perform these steps for each worker node in Kubernetes cluster to prepare your environment for installing the CSI (Container Storage Interface) driver.
+Complete these steps for each worker node in Kubernetes cluster to prepare your environment for installing the CSI (Container Storage Interface) driver.
 
-1. **For RHEL OS users:** Ensure iSCSI connectivity. If using RHCOS or if the packages are already installed, skip this step and continue to step 2.
-
-2. Configure Linux® multipath devices on the host.
+1. Configure Linux® multipath devices on the host.
 
    **Important:** Be sure to configure each worker with storage connectivity according to your storage system instructions. For more information, find your storage system documentation in [IBM Documentation](http://www.ibm.com/docs/).
 
    **Additional configuration steps for OpenShift® Container Platform users (RHEL and RHCOS).** Other users can continue to step 3.
```

Comment:
since this step number turned from 2 to 1, `Other users can continue to step 3` means non-ocp users would skip step 2.
could we change it to something like `Other users can skip this step`, so we won't have to maintain this numbering reference?

Reply:
updated in PR 415

Reply:
I [see](https://github.com/IBM/ibm-block-csi-driver/blob/17d919e94d523754830ebae450b82734ea9bfcec/docs/content/installation/install_compatibility_requirements.md) - thanks!